src/controller/python/ChipDeviceController-ScriptBinding.cpp

```diff
@@ -666,6 +682,11 @@ ChipError::StorageType pychip_ExpireSessions(chip::Controller::DeviceCommissione
 {
     VerifyOrReturnError((devCtrl != nullptr) && (devCtrl->SessionMgr() != nullptr), CHIP_ERROR_INVALID_ARGUMENT.AsInteger());
 
+    //
+    // Stop any active pairing sessions to this node.
```

Comment:
Does the comment help read code? I expect `devCtrl->StopPairing(nodeId)` to "stop the pairing to the node nodeID".

Maybe rephrase as to why we stop or something else that helps in reading the code by enhancing the actual code line.